server/services/zigbee2mqtt/docker/gladys-z2m-mqtt-container.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "gladys-z2m-mqtt",
+  "Image": "eclipse-mosquitto:latest",
```

Comment:
Please fix mosquitto version, we had a problem once mosquitto did upgrade.

Reply:
Fixed to major 2